custom/enikshay/tasks.py

```diff
@@ -488,6 +487,7 @@ def get_aggregated_scores(self, latest_adherence_date, adherence_schedule_date_s
         )
 
         doses_per_week = self.get_doses_per_week()
+        update['doses_per_week'] = doses_per_week
```

Comment:
this doesn't logically make sense to include in the `get_aggregated_scores` method. Can you move it elsewhere? It won't update all the time if left here because of the return value above.